src/ensembl/src/content/home/components/homepage-app-links/HomepageAppLinks.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import { isEnvironment, Environment } from 'src/shared/helpers/environment';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import useOutsideClick from 'src/shared/hooks/useOutsideClick';
+
+import { fetchDataForLastVisitedObjects } from 'src/content/app/browser/browserActions';
+
+import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
+import {
+  getPreviouslyViewedGenomeBrowserObjects,
+  PreviouslyViewedGenomeBrowserObjects
+} from 'src/content/home/homePageSelectors';
+
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as EntityViewerIcon } from 'static/img/launchbar/entity-viewer.svg';
+import { ReactComponent as HomeIcon } from 'static/img/header/home.svg';
+
+import styles from './HomepageAppLinks.scss';
```

Comment:
Should this be moved to the end?

Reply:
Have we established the priority of imports of types vs styles? I've been importing types last; but can do it the other way around.

Reply:
This is the first time I am noticing styles not being the last import as far as I can recall from reviewing your PRs.

Reply:
🤔 
I'll swap them around and try to keep to importing the styles last then.

Reply:
✔️